app/presenters/licence_details_presenter.rb

```diff
@@ -54,6 +68,10 @@ def authority
 
 private
 
+  def has_actions?
+    authority.present? && authority["actions"].present?
+  end
```

Comment:
Can we drop this?  There's `has_any_actions?` defined above which has basically the same method body.

Reply:
Done